app/actions.py

```diff
@@ -84,9 +84,33 @@ def performDeleteRange(self, upperRow, upperCol, lowerRow, lowerCol):
       self.getText(upperRow, upperCol, lowerRow, lowerCol)))
     self.redo()
 
+  def getBookmarkColor(self):
```

Comment:
Does this work well in an 8 color terminal? Heh, well, maybe not work well, but does it not crash and not render super badly (other than how bad things look with only 8 colors)?